### PR TITLE
rotating-cube

### PR DESCRIPTION
**Pull request by alichraghi**
- [x] By selecting this checkbox, I agree to license my contributions to this project under the license(s) described in the LICENSE file, and I have the right to do so or have received permission to do so by an employer or client I am producing work for whom has this right.

### CODE REVIEW

File: examples/rotating-cube/cube_mesh.zig [hunk 1]
```diff
@@ -0,0 +1,49 @@
+pub const Vertex = extern struct {
```
Comment (slimsag) [inline examples/rotating-cube/cube_mesh.zig:1]: FYI I am against starting to duplicate all example code in this repository, that is not a good approach. We should start testing dusk against mach-core instead.

**Comment by slimsag** [comment]
@alichraghi I saw you marked the PR as ready for review - but `examples/zmath.zig`, `examples/rotating-cube` etc. need to be removed as I mentioned

**Comment by alichraghi** [comment]
We don't have a workaround yet. unfortunately that's the only way until the package manager issue get fixed. even after that it would be super hard to develop without ezpkg or relative path's in package manager

**Comment by pdoane** [comment]
> We don't have a workaround yet. unfortunately that's the only way until the package manager issue get fixed. even after that it would be super hard to develop without ezpkg or relative path's in package manager

I've been effectively working out of mach-core (outside of actual build issues).  Agree with @slimsag that it's better to avoid the duplication.

**Review by slimsag — APPROVED** [review]

**Pull request merged**